charts/prometheus-yet-another-cloudwatch-exporter/values.yaml

```diff
@@ -0,0 +1,234 @@
+# Default values for yet-another-cloudwatch-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  registry: ghcr.io
+  repository: nerdswords/yet-another-cloudwatch-exporter
```

Comment:
is this up2date?

Reply:
The new repo default should be `quay.io/prometheuscommunity/yet-another-cloudwatch-exporter`.